runtime/onert/backend/cpu/ops/RoPELayer.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_CPU_OPS_ROPE_LAYER_H__
+#define __ONERT_BACKEND_CPU_OPS_ROPE_LAYER_H__
+
+#include <backend/IPortableTensor.h>
+#include "OperationUtils.h"
+
+#include <exec/IFunction.h>
+
+namespace nnfw
+{
+namespace cker
+{
+class RoPE;
+}
+} // namespace nnfw
+
+namespace onert
+{
+namespace backend
+{
+namespace cpu
+{
+namespace ops
+{
+
+class RoPELayer : public ::onert::exec::IFunction
+{
+public:
+  RoPELayer();
+  ~RoPELayer();
+
+public:
+  template <typename T> void rope();
```

Comment:
How about making this function as private? Because it will be called only by `run()` function`. 

```suggestion
  template <typename T> void rope();
```

Reply:
I agree your comment
i will change private function